src/gui/plugins/modules/EntityContextMenu.cc

```diff
@@ -129,11 +174,28 @@ EntityContextMenu::EntityContextMenu()
 
   // For paste service requests
   this->dataPtr->pasteService = "/gui/paste";
+
+  this->dataPtr->trackPub =
+    this->dataPtr->node.Advertise<msgs::CameraTrack>(this->dataPtr->trackTopic);
+
 }
 
 /////////////////////////////////////////////////
 EntityContextMenu::~EntityContextMenu() = default;
 
+/////////////////////////////////////////////////
+void EntityContextMenu::SetFollowingTarget(const bool &_followingTarget)
```

Comment:
We don't use `const` with primitive types

```suggestion
void EntityContextMenu::SetFollowingTarget(bool &_followingTarget)
```